packages/components/src/index.ts

```diff
@@ -217,6 +217,7 @@ export {
 } from './higher-order/with-focus-return';
 export { default as withNotices } from './higher-order/with-notices';
 export { default as withSpokenMessages } from './higher-order/with-spoken-messages';
+export { ProgressBar } from './progress-bar';
```

Comment:
This isn't really a high-order component, can you move the export above, where the rest of the regular components are? Another thing to keep in mind is that we usually maintain alphabetical order with exports here.

Reply:
Ah, I didn't notice that this section was about HOCs. Will move.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/61062/commits/5ca947a879a9d0976fbd00d50728dc037c37ea27.